packages/presentation/lib/src/pages/movie_details_page/widgets/reviews_widget.dart

```diff
@@ -0,0 +1,165 @@
+import 'package:expandable/expandable.dart';
+import 'package:flutter/material.dart';
+import 'package:presentation/const/app.dart';
+import 'package:presentation/src/common_widgets/image_error_widget.dart';
+import 'package:presentation/src/pages/home_page/widgets/movie_stars.dart';
+import 'package:presentation/src/pages/movie_details_page/model/review_messages_ui.dart';
+import 'package:presentation/src/pages/movie_details_page/widgets/review_list_item_shadow.dart';
+import 'package:presentation/style/colors.dart';
+import 'package:presentation/style/dimens.dart';
+import 'package:presentation/style/text_styles/styles.dart';
+
+import 'cast_and_crew.dart';
+
+class ReviewsWidget extends StatelessWidget {
+  const ReviewsWidget({
+    required this.reviews,
+    required this.isLoading,
+    super.key,
+  });
+
+  final List<ReviewMessageUi> reviews;
+  final bool isLoading;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.all(Dimens.size18),
+      width: double.infinity,
+      height: MediaQuery.of(context).size.height - Dimens.size250,
+      child: isLoading
+          ? ListView(
+              children: List.generate(
+                AppConst.shadowMovieListLength,
+                (index) => ReviewListItemShadow(),
+              ),
```

Comment:
I think it's better to use ListView.builder as you will be using one widget and you won't need a list generate

Reply:
seems u r right, better to use builder, cuz it lazily creates items, and only which should to be displayed while scrolling